### PR TITLE
fix(web): add events and style to icons

### PR DESCRIPTION
**Pull request by issmail-basel**

### CODE REVIEW

File: web/src/components/atoms/Icon/icons.ts [hunk 1]
```diff
@@ -44,4 +54,13 @@ export default {
   plus: PlusOutlined,
   close: CloseOutlined,
   exclamationCircle: ExclamationCircleOutlined,
+  textT: Text,
```
Comment (KaWaite) [inline web/src/components/atoms/Icon/icons.ts:57]: textT?

File: web/src/components/atoms/Icon/icons.ts [hunk 2]
```diff
@@ -44,4 +54,13 @@ export default {
...
+  textAlignLeft: TextAlignLeft,
+  markDown: MarkDown,
```
Comment (KaWaite) [inline web/src/components/atoms/Icon/icons.ts:59]: markdown

**Review by KaWaite — APPROVED** [review]
Just a couple spelling changes

File: web/src/components/atoms/Icon/icons.ts (hunk 1 above)
Comment (issmail-basel) [reply web/src/components/atoms/Icon/icons.ts:57]: Because it has `T` shape, same name Red used in design file.

**Pull request merged**